test/testcommon/src/main/java/io/pravega/test/common/TestUtils.java

```diff
@@ -22,8 +23,11 @@
 public class TestUtils {
     // Linux uses ports from range 32768 - 61000.
     private static final int BASE_PORT = 32768;
-    private static final int MAX_PORT_COUNT = 28232;
-    private static final AtomicInteger NEXT_PORT = new AtomicInteger(1);
+    private static final int MAX_PORT_COUNT = 28233;
+
+    // We use a random start position here to avoid ports conflicts when this method is executed from multiple processes
+    // in parallel. This is needed since the processes will contend for the same port sequence.
+    private static final AtomicInteger NEXT_PORT = new AtomicInteger(new Random().nextInt(MAX_PORT_COUNT));
```

Comment:
This implies two test tasks e.g: :client:test and :controller:test running as different processes can fall in the same range, with a low probability.